### PR TITLE
Merchant order

### PR DESCRIPTION
**Pull request by hashmaster3k**
User stories 49-50

**User Story 49, Merchant sees an order show page**

As a merchant employee
When I visit an order show page from my dashboard
I see the recipients name and address that was used to create this order
I only see the items in the order that are being purchased from my merchant
I do not see any items in the order being purchased from other merchants
For each item, I see the following information:
- the name of the item, which is a link to my item's show page
- an image of the item
- my price for the item
- the quantity the user wants to purchase

**User Story 50, Merchant fulfills part of an order**

As a merchant employee
When I visit an order show page from my dashboard
For each item of mine in the order
If the user's desired quantity is equal to or less than my current inventory quantity for that item
And I have not already "fulfilled" that item:
- Then I see a button or link to "fulfill" that item
- When I click on that link or button I am returned to the order show page
- I see the item is now fulfilled
- I also see a flash message indicating that I have fulfilled that item
- the item's inventory quantity is permanently reduced by the user's desired quantity

If I have already fulfilled this item, I see text indicating such.

**User Story 51, Merchant cannot fulfill an order due to lack of inventory**

As a merchant employee
When I visit an order show page from my dashboard
For each item of mine in the order
If the user's desired quantity is greater than my current inventory quantity for that item
Then I do not see a "fulfill" button or link
Instead I see a notice next to the item indicating I cannot fulfill this item

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -156,3 +156,8 @@ td, th {
 tr:nth-child(even) {
   background-color: #F0F8FF;
 }
+
+.item-image {
+  width: 150px;
+  height: 100px;
+}
```
Comment (hashmaster3k) [inline app/assets/stylesheets/application.css:163]: Sized images so in the order show view page the images aren't huge

File: app/models/order.rb [hunk 2]
```diff
@@ -22,4 +22,8 @@ def change_status_packaged
       update_attribute(:order_status, 'packaged')
     end
   end
+
+  def merchant_items(merch_id)
+    items.select("items.id, items.name, items.image, items.price, items.inventory, item_orders.order_status, item_orders.quantity, item_orders.id as io_id").joins(:item_orders).where(merchant_id: merch_id)
+  end
```
Comment (hashmaster3k) [inline app/models/order.rb:28]: Joins the two tables to get only the information needed as opposed to all columns from both tables. Renamed item_orders.id to io_id to not confused with item.id

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (dcoleman21) [reply app/assets/stylesheets/application.css:163]: This looks much better.

**Review by dcoleman21 — APPROVED** [review]
This looks good and the images are way better to look at.

**Pull request merged**